### DISCUSSION TITLE
<404 Error file not found>

### DISCUSSION THREAD

**Discussion by toluhenok** [community/community#47786] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

It has been over 24 hours since I published my web pages on GitHub pages, but I still get 404 File not found. 
Here is the page: https://toluhenok.github.io/dataportfolio/

I need help

**Comment by rilysh** [comment, marked as answer]
@toluhenok The jekyll site [toluhenok.github.io/dataportfolio]( https://toluhenok.github.io/dataportfolio/) was failed to build, see [here](https://github.com/toluhenok/dataportfolio/actions/runs/4205362932/jobs/7297339574)
 
I also noticed, you didn't pass TLD in your CNAME file, which you deleted, see [here](https://github.com/toluhenok/dataportfolio/commit/5ccc7c2dfaa59d6f0e5dc7a5305d387c844cc2ba)

First, rebuild your site on your local machine, see what the error is. Make sure you've removed your previous domain URL (if you added it) from the custom domain and rebuild it. If everything works, your site should be available at [https://toluhenok.github.io/dataportfolio](https://toluhenok.github.io/dataportfolio/)

Hope this helps.

**Reply by toluhenok** [reply]
Thank you very much. The site works now
